### PR TITLE
New: v8.0.0 blog post

### PR DESCRIPTION
**Pull request by nzakas**
Announcement of upcoming changes in v8.0.0. I purposely stuck with the most high-profile breaking changes, as we can dig into details of everything when we create our upgrade guide.

### CODE REVIEW

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+We want to share with you the significant changes that will be coming in v8.0.0 so you can better prepare for the changes and understand why the changes are necessary.
+
+## Significant changes in v8.0.0
+
+The following changes are important to understand as we move towards v8.0.0.
+
+### Dropping support for Node.js v10.x and 13.x
+
+Node.js v10.x end-of-life was April 30, 2021 and Node.js v13.x end-of-life was June 1, 2021. As a result, ESLint v8.0.0 [removes support](https://github.com/eslint/eslint/issues/14023) for both of these Node.js versions.
```
Comment (mdjermanovic) [inline _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:20]:
```suggestion
Node.js v10.x end-of-life was April 30, 2021 and Node.js v13.x end-of-life was June 1, 2020. As a result, ESLint v8.0.0 [removes support](https://github.com/eslint/eslint/issues/14023) for both of these Node.js versions.
```

Also, per https://github.com/eslint/eslint/pull/14592 we will remove support for v15.x too?

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Support for ES2022 [comes to ESLint v8.0.0](https://github.com/eslint/eslint/issues/14343) with native support for class fields and top-level `await`.
+
+### Rules with suggestions now require the `meta.hasSuggestions` property
+
+Beginning in ESLint v8.0.0, rules that provide suggestions [must include a `meta.hasSuggestions: true` property](https://github.com/eslint/eslint/issues/14312). This allows ESLint to understand what a rule's capabilities are just by inspecting the rule object. Rules that provide suggestions without `meta.hasSuggestions: true` will cause an error.
+
+### Fixable rules now require `meta.fixable` property
+
+In ESLint v7.0.0 and earlier, it was possible to write a rule as a function instead of an object and to provide fixes without specifying a `meta.fixable` property. Beginning in v8.0.0, ESLint requires all rules that provide fixes to be specified in object format and [include `meta.fixable: true`](https://github.com/eslint/eslint/issues/13349).
```
Comment (mdjermanovic) [inline _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:36]: Not `true`, per the [specification](https://eslint.org/docs/developer-guide/working-with-rules#rule-basics) it should be string value `"code"` or `"whitespace"`.

Though, we're not validating the exact value - the only validation is that it must be truthy.

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### Stricter rule schema validation
+
+With an [upgrade to Ajv 8](https://github.com/eslint/eslint/issues/13888), ESLint v8.0.0 will have improved error detection for rule schemas. There were several areas where the validation would fail silently with Ajv 6, which could lead to end-user errors that were hard to detect. This stricter validation will help catch more errors earlier in the rule development process.
+
+### Removal of undocumented APIs; removal of `CLIEngine`
+
+ESLint v8.0.0 [defines an `exports` field](https://github.com/eslint/eslint/issues/13654) to remove access to undocumented internal APIs that some packages were accessing.
+
+As part of this change, we have removed the deprecated `CLIEngine` class to encourage adoption of the replacement `ESLint` class. `CLIEngine` was [deprecated in 2020]() and we can't continue to support it as we move forward with more ambitious features for ESLint.
```
Comment (mdjermanovic) [inline _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:54]: missing url in the link

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md (hunk 1 above)
Comment (nzakas) [reply _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:20]: Good point, I’ll add that.

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md (hunk 2 above)
Comment (nzakas) [reply _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:36]: I keep forgetting that. :) I’ll clarify.

File: _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md (hunk 3 above)
Comment (nzakas) [reply _posts/2021-06-08-whats-coming-in-eslint-8.0.0.md:54]: Thanks.

**Pull request merged**